contributing/tools-maintenance.rst

```diff
@@ -2,7 +2,7 @@ Development tools: maintenance
 ==============================
 
 Most of the development tools used internally by the OME project are
-Python-based and hosted on the :pypi:`Python Package Index <>`. Their source
+Python-based and hosted on `PyPI`_. Their source
```

Comment:
The backquotes are unnecessary here and below: `hosted on Pypi_.` should work.

Reply:
other links could be updated too
but I won't fix that in this PR